FE/src/components/Movies/Movie.tsx

```diff
@@ -0,0 +1,18 @@
+import { MovieType } from './MovieTypes';
+
+type MovieProps = {
+  movie: MovieType,
+};
+
+const Movie = (props: MovieProps) => {
+  const { movie } = props;
+  return (
+    <>
+      <div key={movie._id}>

```

Comment:
The key is not needed here, key is needed in the map, where you render the `<Movie />` component.

Reply:
I know, it was left there by mistake, deleted.